extensions/amp-list/0.1/amp-list.js

```diff
@@ -458,8 +458,12 @@ export class AmpList extends AMP.BaseElement {
     }
 
     return fetch.catch(error => {
+      const event = error
+        ? createCustomEvent(this.win, `${TAG}.error`,
+            dict({'response': error.response}))
+        : null;
       const actions = Services.actionServiceForDoc(this.element);
-      actions.trigger(this.element, 'fetch-error', null, ActionTrust.LOW);
+      actions.trigger(this.element, 'fetch-error', event, ActionTrust.LOW);
```

Comment:
Need this change in `amp-form.js` too?

Reply:
Yes, having `fetch-error` available for `amp-form` would be good for consistency.

Reply:
Disregard my previous comment, `submit-error` already exists and should be just fine.

Reply:
Sorry, I think I meant amp-state.js. /jetlag